internal/budgetadjustments/get.go

```diff
@@ -0,0 +1,131 @@
+package budgetadjustments
+
+import (
+	_ "embed"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"os"
+	"time"
+
+	"github.com/nobl9/nobl9-go/sdk"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/nobl9/sloctl/internal/printer"
+)
+
+type GetCmd struct {
+	client           *sdk.Client
+	outputFormat     string
+	fieldSeparator   string
+	recordSeparator  string
+	out              io.Writer
+	adjustment       string
+	from, to         TimeValue
+	project, sloName string
+}
+
+type SLO struct {
+	Project string `json:"project" validate:"required"`
+	Name    string `json:"name"    validate:"required"`
+}
+
+type Event struct {
+	EventStart time.Time `json:"eventStart"`
+	EventEnd   time.Time `json:"eventEnd"`
+	Slos       []SLO     `json:"slos"`
+}
+
+//go:embed examples/get_example.sh
+var example string
+
+func NewGetCmd(client *sdk.Client) *cobra.Command {
+	get := &GetCmd{out: os.Stdout}
+
+	cmd := &cobra.Command{
+		Use:   "get",
+		Short: "Return a list of events for given Adjustment with related SLOs.",
+		Long: "This endpoint only return past and ongoing events (events that are already started)." +
+			"Please see Editing budget adjustments." +
+			"Maximum 500 events can be returned." +
+			"Optional filtering for specific SLO (only one). If SLO is defined we will return only events" +
```

Comment:
Optional filtering for specific SLO (only one), in such cases both slo-name and slo-project flags are required.

Reply:
lets update it when doc will be ready